packages/node-core/test/sandbox/tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "outDir": "."
   },
   "exclude": [],
-  "include": ["./main.ts"]
+  "include": ["./main.ts", "./project.ts"]
```

Comment:
Will users have to update their ts config if they use a ts manifest?

Reply:
No, it doesn't require user project add this to tsconfig. 

This line was added to build `project.js`, in order for testing purpose.